utility_network/perform-valve-isolation-trace/README.md

```diff
@@ -0,0 +1,53 @@
+# Perform valve isolation trace
+
+Run a filtered trace to locate operable features that will isolate an area from the flow of network resources.
+
+![Image of Perform valve isolation trace](PerformValveIsolationTrace.png)
+
+## Use case
+
+Determine the set of operable features required to stop a network's resource, effectively isolating an area of the network. For example, you can choose to return only accessible and operable valves: ones that are not paved over or rusted shut.
+
+## How to use the sample
+
+Create and set the configuration's filter barriers by selecting a category. Check or "Include isolated features". Click "Trace" to run a subnetwork-based isolation trace.
+
+## How it works
+
+1. Create a `MapView`.
+2. Create and load a `UtilityNetwork` with a feature service URL.
+3. Create a `Map` that contains `FeatureLayer`(s) that are part of this utility network.
+4. Create a default starting location from a given asset type and global id.
```

Comment:
I think these details are left out of here by design, with this sample focusing more on the 'isolation trace' functionality rather than the shared set-up for a UN trace. I have to admit it's a long time ago now so I can't remember the rationale!
@jenmerritt @sclaridge @Rachael-E if you are all happy with adding more detail, that can be done! Since you are the resident sample experts, I'd follow your advice here. I myself prefer to stick as closly to the common design as possible, but that's probably not always the best way forward.

Reply:
@JonLavi if there are some easy code snippets that could be fitted in here to deal with this then I'd be in favour of adding that, that small detail is ok to go against the design (is this design still up to date with the common samples after all the updates that were done there?) I've also just noticed in Point 3 it says: 

"Create a `Map`"

That should be Create an `ArcGISMap`.